### PR TITLE
Replacing non US-ASCII characters

### PR DESCRIPTION
**Pull request by aquister**
Puppet fails due to to non US-ASCII characters during acceptance testing on different platforms and puppet versions.

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -18,4 +18,37 @@
   $hostname = undef
   $max_message_size = 65536
   $logrotate = true
+
+  case $::operatingsystem {
```
Comment (cristifalcas) [inline manifests/params.pp:22]: Can you use here osfamily? Or it's not the same for debian/ubuntu

**Comment by cristifalcas** [comment]
This is huge :). Thank you for your work. 

Can you resolve the conflicts also?

File: manifests/service.pp [hunk 2]
```diff
@@ -1,5 +1,41 @@
 # Takes care of starting heka service
 class heka::service {
+
+  case $heka::service_provider {
```
Comment (cristifalcas) [inline manifests/service.pp:4]: I consider managing files and directories as part of the config class. Any objections to move this block to config.pp?

File: manifests/params.pp (hunk 1 above)
Comment (aquister) [inline manifests/params.pp:22]: The check for $::operatingsystemmajrelease is different for Debian and Ubuntu

File: manifests/service.pp (hunk 2 above)
Comment (aquister) [inline manifests/service.pp:4]: Done!

**Pull request closed without merging**